ClimsoftVer4/ClimsoftVer4/formImports.vb

```diff
@@ -1,4 +1,19 @@
-﻿Public Class formImports
+﻿' CLIMSOFT - Climate Database Management System
+' Copyright (C) 2015
```

Comment:
@tmundoga, @mhabimana - when you update the licence information could you update the Copyright year from `2015` to `2017`?